lib/MT/Util.pm

```diff
@@ -700,9 +700,9 @@ sub _change_lf {
 sub html_text_transform {
     my $str = shift;
     $str = '' unless defined $str;
-    my $tags = qr!(?:h1|h2|h3|h4|h5|h6|table|ol|dl|ul|li|menu|dir|p|pre|center|form|fieldset|select|blockquote|address|div|hr|script|style)!;
+    my $tags = qr!(?:h1|h2|h3|h4|h5|h6|table|ol|dl|ul|li|menu|dir|p|pre|center|form|fieldset|select|blockquote|address|div|hr|script|style|article|aside|details|dialog|figcaption|figure|footer|header|hgroup|main|map|nav|section|template|thead|tfoot|tbody|tr|th|td|caption|colgroup|col|dt|dd|optgroup|legend|menuitem|option|param|summary|area|svg)!;
```

Comment:
I think `svg` should not be included in the `$tag`. Because the browser renders `svg` as an inline element.

Reply:
I think
```html
a
<svg viewBox="0 0 10 10" xmlns="http://www.w3.org/2000/svg" stroke="red" fill="grey">
  <circle cx="5" cy="5" r="10" />
</svg>
b
```
should be
```html
a<br />
<svg viewBox="0 0 10 10" xmlns="http://www.w3.org/2000/svg" stroke="red" fill="grey">
  <circle cx="5" cy="5" r="10" />
</svg><br />
b<br />
```

Reply:
thanks. I wonder if people really want to wrap multi-lined svg with p, but the point is taken.